hyperion-timber/src/main/java/com/willowtreeapps/hyperion/timber/list/TimberLogListActivity.java

```diff
@@ -54,4 +61,18 @@ public boolean onSupportNavigateUp() {
         throw new RuntimeException("TimberLogTree not planted in forest.");
     }
 
-}
\ No newline at end of file
+    @Override
+    public void onClick(View view) {
```

Comment:
This needs to switch on the view passed in. The logging should be moved to its own method and a switch case added to handle the new button and a default case that probably throws an exception.

Reply:
Done!